plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -93,6 +93,7 @@
     private boolean immutablePartitions;
     private Optional<InsertExistingPartitionsBehavior> insertExistingPartitionsBehavior = Optional.empty();
     private boolean createEmptyBucketFiles;
+    private boolean deleteSchemaLocationsFallback = true;
```

Comment:
Why `true` as default. You probably talked this through.

Reply:
It's the current behavior, so it's less likely to cause problems for anyone upgrading. But it's also the less-safe behavior for someone who isn't familiar with Trino who might , especially if they aren't very familiar with it. I think we did discussed it last week, but maybe we should change it?

What do you think, @findepi?

Reply:
The point of the change is "do not delete those random files when deleting schema directory".
`false` feels more appropriate in case we cannot verify whether there are any files in there.
let's also add code comment discussion the rationale behind the choice we put in the default

Reply:
Switched the default and added a comment.